slam-ws/src/nav_husky_gazebo/config/base_local_planner_params.yaml

```diff
@@ -1,3 +1,11 @@
+# Navigation Function (navigation path)
+NavfnROS:
+  # Specifies whether or not to allow navfn to create plans that traverse unknown space.
+  allow_unknown: true
+  # A tolerance on the goal point for the planner.
+  default_tolerance: 0.1
+
+# Plans robot movement [I think]
```

Comment:
Looks like this restricts the motion along the chosen path? This will be very useful for later since we will be constrained via speed limits.

Reply:
Shouldn't this be a comment and not a review?